golang/gometalinter.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Deadline": "120s",
+    "Disable": [
+        "gas"
```

Comment:
Also disable gotype here it has alot of false positives

Reply:
Cool, done. I was using lib-go as a testbed for this, which didn't disable `gotype`.